index.html

```diff
@@ -0,0 +1,12 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title></title>
+  </head>
+  <body>
+    <div style="color:blue;">hello world</div>
```

Comment:
- [ ] **[REQUIRED CHANGES]** Avoid using in-line stying as much as possible! They make our code too extensive, and can lead to confusion for other programmers that check your code, or even you in the future!
[This article](https://www.thoughtco.com/avoid-inline-styles-for-css-3466846) does a wonderful job of explaining why we should use a separate file for the styling in our applications and pages!

- [ ] **[REQUIRED CHANGES]** As per project requirements, you need to encapsulate your 'Hello World' inside a `<header>` tag, so the SEO knows that 'Hello World' is part of the header of your website, so it can display the header's information on the SE!

- [ ] **[REQUIRED CHANGES]** As per project requirements, you also need to encapsulate your 'Hello World' inside the `<h1>` tag. This is good for the SEO of the website, as it would help SE to find what is relevant to display for the users!

___

- [ ] **[OPTIONAL]** After applying all of the required changes, feel free to remove the `<div>` if you're not going to customize it. In this scenario, since your `<header>` only contains the `<h1>` title, a `<div>` wouldn't be needed. As it holds no semantic value and you could style the `<header>` directly.
But remember, this is optional! And if you're going to use it later on, feel free to leave it like this! 😊